resources/register-jaas-account.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+set -e
+
+EMAIL=$1
+DOMAIN=$2
+
+if [ -z "${DOMAIN}" ] || [ -z "${EMAIL}" ]; then
+    echo "You need to provide email and domain as parameters."
+    exit 1
+fi
+
+JAAS_ENDPOINT="https://account-provisioning.cloudflare.jitsi.net/operations"
+CHALLENGE_FILE="/usr/share/jitsi-meet/jitsi-challenge.txt"
+SUPPORT_MSG="Reach out to JaaS support or retry with /usr/share/jitsi-meet/scripts/register-jaas-account.sh"
+
+create_error=0
+create_data=$(curl -s -f -X 'POST' "${JAAS_ENDPOINT}" -H 'Content-Type: application/json' -H 'accept: */*' \
+  -d "{ \"domain\": \"${DOMAIN}\", \"email\": \"${EMAIL}\" }") || create_error=$?
+if [ ${create_error} -ne 0 ]; then
+    echo "Account creation failed. Status: ${create_error}"
```

Comment:
Can the error be read from the response payload?

Reply:
Yep, I will add and that to the print.